librad/src/keys.rs

```diff
@@ -59,27 +49,27 @@ pub trait SignError: error::Error + Send + Sync + 'static {}
 impl<T: error::Error + Send + Sync + 'static> SignError for T {}
 
 /// A device-specific signing key
-#[derive(Clone, Eq, PartialEq)]
+#[derive(Copy, Clone)]
 #[cfg_attr(test, derive(Debug))]
-pub struct SecretKey(ed25519::SecretKey);
+pub struct SecretKey(ed25519::SigningKey);
 
 /// The public part of a `Key``
 #[derive(Clone, Debug, Eq, PartialEq, PartialOrd, Ord, Hash)]
-pub struct PublicKey(ed25519::PublicKey);
+pub struct PublicKey(ed25519::VerificationKeyBytes);
 
 impl From<sign::PublicKey> for PublicKey {
     fn from(other: sign::PublicKey) -> PublicKey {
-        PublicKey(ed25519::PublicKey(other.0))
+        PublicKey(ed25519::VerificationKeyBytes::from(other.0))
     }
 }
 
 /// A signature produced by `Key::sign`
-#[derive(Clone, Debug, Eq, PartialEq, PartialOrd, Ord)]
+#[derive(Clone, Debug, Eq, PartialEq)]
```

Comment:
We don't use these instances in this crate, and they aren't available on ed25519-zebra. If we use them elsewhere, we'd have to patch upstream.